Sofa/Component/Compat/src/SofaBaseCollision/DefaultPipeline.h

```diff
@@ -21,12 +21,12 @@
 ******************************************************************************/
 #pragma once
 
-#include <sofa/component/collision/detection/algorithm/DefaultPipeline.h>
+#include <sofa/component/collision/detection/algorithm/CollisionPipeline.h>
 
-SOFA_DEPRECATED_HEADER("v22.06", "v23.06", "sofa/component/collision/detection/algorithm/DefaultPipeline.h")
+SOFA_DEPRECATED_HEADER("v22.06", "v23.06", "sofa/component/collision/detection/algorithm/CollisionPipeline.h")
 
 namespace sofa::component::collision
 {
-    using DefaultPipeline = sofa::component::collision::detection::algorithm::DefaultPipeline;
+    using DefaultPipeline = sofa::component::collision::detection::algorithm::CollisionPipeline;
 
 } // namespace sofa::component::collision
```

Comment:
You are missing the compat with v22.12, where you would need to add an alias from the (ng) namespace and the old name `sofa::component::collision::detection::algorithm::DefaultPipeline` (from v22.12) 😗
something like that 
```
namespace sofa::component::collision::detection::algorithm
{
    using DefaultPipeline = sofa::component::collision::detection::algorithm::CollisionPipeline;
}
```